deployment/ansible/files/peer_keys/13.250.162.250.pub

```diff
@@ -0,0 +1,7 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
we don't need to encrypt this - the pubkeys (it's just additional noise in the playbooks)

Reply:
true...